internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -266,13 +268,21 @@ func (q amsQuotaService) ReserveQuota(kafka *dbapi.KafkaRequest) (string, *error
 	// will be empty if no marketplace account is used
 	rr.BillingMarketplaceAccount(kafka.BillingCloudAccountId)
 
+	// TODO: temporary workaround. This is to be removed when this issue is implemented: https://issues.redhat.com/browse/SDB-3157
+	getClusterId := func(kafka *dbapi.KafkaRequest) string {
+		if kafka.DesiredKafkaBillingModel == "eval" {
+			return kafkaId + "-eval"
```

Comment:
this can be rewritten as `return fmt.Sprintf("%s-eval", kafkaId)`

Reply:
Yeah, but in this case I find concatenating the string more readable.